### PR TITLE
Add Vision framework/Lint fixes

### PR DESCRIPTION
**Pull request by frets1700**

### CODE REVIEW

File: Sources/Controllers/BarcodeScannerViewController.swift [hunk 1]
```diff
@@ -32,308 +33,299 @@ public protocol BarcodeScannerDismissalDelegate: class {
...
+    public var symbologies = VNDetectBarcodesRequest.supportedSymbologies {
+        didSet {
+            cameraViewController.symbologies = symbologies
+        }
+    }
+
+    // MARK: - Private properties
+
+    /// Flag to lock session from capturing.
+    private var locked = false
```
Comment (augustoguido) [inline Sources/Controllers/BarcodeScannerViewController.swift]: could rename to `isLocked` and maybe `areConstraintsActivated`

Comment (august10) [inline Sources/Controllers/BarcodeScannerViewController.swift:63]: could be named `isLocked` and maybe `areConstraintsActivated`

File: Sources/Controllers/CameraViewController.swift [hunk 2]
```diff
@@ -1,439 +1,498 @@
...
-                             didOutput metadataObjects: [AVMetadataObject],
-                             from connection: AVCaptureConnection) {
-    delegate?.cameraViewController(self, didOutput: metadataObjects)
-  }
+extension CameraViewController {
+    func processClassification(_ request: VNRequest) {
+        guard let barcodes = request.results else { return }
+        DispatchQueue.main.async {[weak self] in
+            guard let self = self else {
+                /// FIXME: Show error
```
Comment (august10) [inline Sources/Controllers/CameraViewController.swift:479]: just a reminder that there's something pending here :)

**Review by august10 — APPROVED** [review]
nice 👍

File: Sources/Controllers/CameraViewController.swift (hunk 2 above)
Comment (augustoguido) [inline Sources/Controllers/CameraViewController.swift:479]: Pending FIXME comment

**Review by augustoguido — APPROVED** [review]

**Pull request merged**